Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -70,20 +74,25 @@ public void Generate(string type, string baseWritePath, bool isEquatable, List<A
 
             foreach (var templatePath in templatePaths)
             {
-                var templateNameStartIndex = templatePath.LastIndexOf(Path.DirectorySeparatorChar) + 1;
+#if UNITY_2018_4_OR_NEWER
```

Comment:
Same as above + add UNITY_2019_1_OR_NEWER condition. The UNITY_2019_1_OR_NEWER condition should be before UNITY_2018_4_OR_NEWER, the second if-statement will never be used when running Unity 2019 since the UNITY_2018_4_OR_NEWER flag will also be true in this case.